src/Microsoft.Identity.Web/InstanceDiscovery/IssuerConfigurationRetriever.cs

```diff
@@ -40,7 +42,7 @@ public async Task<IssuerMetadata> GetConfigurationAsync(string address, IDocumen
             };
 
             string doc = await retriever.GetDocumentAsync(address, cancel).ConfigureAwait(false);
-            return JsonSerializer.Deserialize<IssuerMetadata>(doc, options);
+            return JsonSerializer.Deserialize<IssuerMetadata>(doc, options)!;
```

Comment:
This case is weird - the compiler says `Deserialize` can return null, but the docs don't mention it, say it just throws exceptions.

Reply:
want to make a comment here then for future reference?

---
In reply to: [444034173](https://github.com/AzureAD/microsoft-identity-web/pull/231#discussion_r444034173) [](ancestors = 444034173)

Reply:
Added.